### ISSUE TITLE
Css 

### ISSUE THREAD

**Issue by sheri-thomson** [sheri-thomson/responsive-icon-cards#1]
@thomasjbradley

Hello - I'm a little confused as to what css needs to be applied previous to the media query section. Am I targeting classes there and in each media query?  At 400 and 1400 screen width the image is supposed to float left and be treated as an inline block but I have them set to block .. can you point me in the right direction please?

**Comment by thomasjbradley** [comment]
Yes, you'll be targeting those classes in each of the media queries to adjust the layout. You don't need to use `float`, but instead, inside a media query make the image `inline-block`.

**Comment by sheri-thomson** [comment]
@thomasjbradley 
Markbot is accepting all screenshots and it looks nothing like what it is supposed to. I've made a real mess of things and over complicated it..now I don 't know if i should just restart or how to clean things up :''(

**Comment by thomasjbradley** [comment]
If all the screenshots are accepted by Markbot just move on to the next project.

My strictness needs to be improved on this assignment—I recognize that people can create completely different layouts and still pass.

**Comment by sheri-thomson** [comment]
@thomasjbradley 
But I literally have overlapping text - it's awful and made me cry lol
AND it's saying I don't have the ```.card-body h2``` and ```.card-body p``` selector..but I do

K - I undid the overlap, I just cant get the image to float left of the body

**Comment by thomasjbradley** [comment]
Just accept the free marks! Don't worry about it—move on.

I see you do have the `.card-body h2` and `.card-body p` selectors inside media queries but not outside.

**Comment by sheri-thomson** [comment]
I undid everything and don't really know how to get back to where I was..that being said, I have all the proper selectors! But now nothing is working.. it's fine I guess ill just start over
